File: LLM-Translations/GPT-4/codenet/Python/C/s554858080.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>

int main() {
    char input[100];
    fgets(input, sizeof(input), stdin);

    char *token = strtok(input, " ");
    char num[100] = "";

    while (token != NULL) {
        strcat(num, token);
        token = strtok(NULL, " ");
    }

    int number = atoi(num);

    if (number % 4 == 0) {
        printf("YES\n");
    } else {
        printf("NO\n");
    }

    return 0;
}
// 

File: LLM-Translations/GPT-4/codenet/Python/Go/s554858080.go
package main

import (
	"fmt"
	"strconv"
	"strings"
)

func main() {
	var input string
	fmt.Scanf("%s", &input)

	inputList := strings.Split(input, " ")
	num := strings.Join(inputList, "")

	n, _ := strconv.Atoi(num)

	if n%4 == 0 {
		fmt.Println("YES")
	} else {
		fmt.Println("NO")
	}
}
// 
